src/reflect/scala/reflect/internal/Types.scala

```diff
@@ -2703,7 +2703,7 @@ trait Types
     }
     private def customToString = sym match {
       case RepeatedParamClass | JavaRepeatedParamClass => args.head.toString + "*"
-      case ByNameParamClass   => "=> " + args.head
+      case ByNameParamClass if ! args.isEmpty  => "=> " + args.head
```

Comment:
I've never seen extra space after unary op style, so it looks strange to me, especially since `args.nonEmpty` is available. Also, the arrows for the three cases haven't aligned in years; I would either align them or remove spaces from the default.

Reply:
After looking a bit into it, and since the 3rd case is shorter than the first one, I have opted for adding a newline between the guard and the body of each case.